extensions/markdown-basics/syntaxes/markdown.tmLanguage.json

```diff
@@ -3084,8 +3084,8 @@
 					"name": "punctuation.definition.strikethrough.markdown"
 				}
 			},
-			"match": "(?<!\\\\)(~{2,})((?:[^~]|(?!(?<![~\\\\])\\1(?!~))~)*+)(\\1)",
+			"match": "(?<!\\\\)(~{2,})(?!(?<=\\w~~)_)((?:[^~]|(?!(?<![~\\\\])\\1(?!~))~)*+)(\\1)(?!(?<=_\\1)\\w)",
```

Comment:
Thanks for looking into this. Can you please make the main markdown grammar fixes here: http://github.com/microsoft/vscode-markdown-tm-grammar/

We pull in the grammar from that repo automatically so any changes here would get overwritten

Thanks